book/actix-plugin.md

```diff
@@ -156,6 +156,55 @@ curl http://localhost:8080/api/spec
 
 Similarly, if we were to use other extractors like `web::Query<T>`, `web::Form<T>` or `web::Path`, the plugin will emit the corresponding specification as expected.
 
+#### Manually defining additional response codes
+
+There is a macro `api_v2_errors` which helps to manually add responses other than 200. Here is an example.
+We define an enum that represents the error. To use it in actix-web it needs to implement `Display` and `ResponseError`.
+There is no need to impl Display by hand for such simple enums - we can use `enum-display-derive` lib.
+
+```rust
+use actix_web::{
+    error::{ErrorInternalServerError, ErrorUnauthorized, ResponseError},
+    HttpResponse,
+};
+use enum_display_derive::Display;
+use paperclip::actix::api_v2_errors;
+use std::fmt::Display;
+
+#[api_v2_errors(
+    "400 Bad Request",
+    "401 Unauthorized: Can't read session from header",
+    "500 Internal Server Error",
+)]
+#[derive(Debug, Display)]
+pub enum UserError {
+    Unauth,
+    Internal(String),
+}
+
+impl ResponseError for UserError {
```

Comment:
Let's keep this subtle and remove `impl ResponseError for UserError` entirely.

Reply:
So do you think 'examples' directory could be a good place for this in future?

Reply:
Yeah, we'll take care of detailed examples in the future. Right now, this feels redundant.